### PR TITLE
test

### PR DESCRIPTION
**Pull request by rmadiwalid**

### CODE REVIEW

**Comment by rmadiwalid** [comment]
fix tooltip.show called when domNode is null (causing random failing intern tests)

**Comment by wkeese** [comment]
I don't see why that condition would ever occur (and I don't remember intern tests failing because of that).

**Comment by rmadiwalid** [comment]
I meant in my intern tests when I run all tests, the Tooltip tests are randomly failing cause dby "innerHTML of null" I had to patch "dijit/Tooltip" show method like this:

define([
    // load the module that needs patching
    "dijit/Tooltip"
], function(
    Tooltip
) {
    // Keep reference of old show method that needs to be patched

    var oldTooltipShow = Tooltip._MasterTooltip.prototype.show;

    function _getPatchedBehaviour() {
        Tooltip._MasterTooltip.prototype.show = function() {

            if (this.containerNode) {
                return oldTooltipShow.apply(this, arguments);
            }

            return null;
        };
    }

    _getPatchedBehaviour();

    return {
        getOriginalBehaviour: function() {
            Tooltip._MasterTooltip.prototype.show = function() {
                return oldTooltipShow.apply(this, arguments);
            };
        },

        getPatchedBehaviour: _getPatchedBehaviour
    };
});

**Comment by wkeese** [comment]
OK, but it would be good to find out why.   It seems like you are just hiding an error.

File: Tooltip.js [hunk 1]
```diff
@@ -80,6 +80,9 @@ define([
 			if(this.aroundNode && this.aroundNode === aroundNode && this.containerNode.innerHTML == innerHTML){
 				return;
 			}
+			if(!this.containerNode){
+			return;
```
Comment (dylans) [inline Tooltip.js]: incorrect indentation

**Comment by msssk** [comment]
This PR has not been adequately justified. As previously mentioned, it is just hiding an error. In addition it will never work correctly because the check for the existence of `this.containerNode` is performed _after_ `this.containerNode.innerHTML` has already been referenced.

**Pull request closed without merging**